src/mz390.java

```diff
@@ -11638,27 +11639,46 @@ private void exec_pc_pfx_o(){
     	 *   S = macro found in sysmac dir
     	 *   U = unknown
     	 */
+        //System.out.println("afk mz390.exec_pc_pfx_o tot_exp_stk_var=" + tot_exp_stk_var);         // #485 **!!
     	if (tot_exp_stk_var > 0){
 			setc_value1 = get_setc_stack_value().toUpperCase();
-			int opcode_type = find_opcode_type(setc_value1);
+            //System.out.println("afk mz390.exec_pc_pfx_o setc_value1=" + setc_value1);             // #485 **!!
+            int index = 0;                                                                          // #485
+            String key = setc_value1;                                                               // #485
+            index = tz390.find_key_index('R',key);                                                  // #485
+            //System.out.println("afk mz390.exec_pc_pfx_o OPSYN index=" + index);                   // #485 **!!
+            if (index >= 0) // OPSYN defined:                                                       // #485
+               {if (tz390.opsyn_old_name[index] == null) // OPSYN deletes mnemonic                  // #485
+                   {key = "  Undefined Mnemonic  ";                                                 // #485
+                    }                                                                               // #485
+                else // OPSYN defines alternate mnemonic                                            // #485
+                   {key = tz390.opsyn_old_name[index];                                              // #485
+                    }                                                                               // #485
+                //System.out.println("afk mz390.exec_pc_pfx_o Base mnemonic=" + key);               // #485 **!!
+                }                                                                                   // #485
+            int opcode_type = find_opcode_type(key);                                                // #485
+            //System.out.println("afk mz390.exec_pc_pfx_o opcode_type=" + opcode_type);             // #485 **!!
 			if (opcode_type >= 0){
 				if (opcode_type <= tz390.max_ins_type){
-					if (opcode_type == 3
-							|| opcode_type == 6
-							|| opcode_type == 13){
-						setc_value = "E"; // BRX, BCX, BRCX extended mnemonic
-					} else {
-						setc_value = "O"; // machine opcode
-					}
+                    index = tz390.find_key_index('O',key);                                          // #485
+                    //System.out.println("afk mz390.exec_pc_pfx_o OPCODE index=" + index);          // #485 **!!
+                    if (index > -1)                                                                 // #485
```

Comment:
Suggestion only. Nothing wrong. For consistency with previous checks after tz390.find_key_index() invocations, replace "index > -1" with "index >= 0".